web-common/src/features/dashboards/time-series/DimensionValueMouseover.svelte

```diff
@@ -52,18 +62,18 @@
 
   $: points = yValues
     .map((dimension) => {
-      const y = dimension.y;
+      const y = Number(dimension.y);
```

Comment:
`Number(null)` casts y as `0` which is not correct. This makes `currentPointIsNull` always false. We need to preserve the value of y as `null`